packages/app-shortcut/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@capawesome/capacitor-app-shortcut",
-  "version": "5.0.0",
+  "version": "6.0.0",
```

```suggestion
  "version": "5.0.0",
```

The version is automatically incremented by GitHub Actions upon release.